qwandaq/src/main/java/life/genny/qwandaq/utils/BaseEntityUtils.java

```diff
@@ -109,7 +106,18 @@ public PCM getPCM(String code) {
 	 * @return
 	 */
 	public Definition getDefinition(String code) {
-		return Definition.from(getBaseEntity(code, true));
+		return Definition.from(getDefinition(code, true));
```

Comment:
This call to Definition.from is redundant overhead. It is called already on Line 120 (getDefinition(String code, boolean bundleAttributes)) so we are calling it a second time before returning here